res/styles/style-index.css

```diff
@@ -4,16 +4,16 @@ h1 {
 
 .container {
     display: grid;
-    grid-template-columns: repeat(9, 9.5vw);
-    grid-template-rows: repeat(3, 12.5vw);
-    grid-gap: 3px;
+    grid-template-columns: repeat(9, 100px);
+    /*grid-template-rows: repeat(3, 9.5vw12.5vw);
```

Comment:
Remove comments you have them anyway in the revision of git, if you need them in the future. Only leave comments with explanation of the current code inside.

Reply:
Commented codes removed